src/test/java/core/basesyntax/WorkWithFileTest.java

```diff
@@ -28,7 +29,11 @@
 
     @Test
     public void getStatisticAboutApple() {
-        workWithFile.getStatistic("apple.csv", APPLE_RESULT_FILE);
+        try {
+            workWithFile.getStatistic("apple.csv", APPLE_RESULT_FILE);
+        } catch (FileNotFoundException e) {
+            System.out.println("Can't write data to file");
```

Comment:
The error message 'Can't write data to file' is not very informative. Consider using a more descriptive message that includes the exception details.